tom/github.py

```diff
@@ -1,6 +1,7 @@
 import random
 import requests
 import re
+import os
```

Comment:
because of
```
os.makedirs('github_repos', exist_ok=True)
```
later in the code.
(the code was broken before this commit - when splitting Tom from one file into several ones - due to lack of proper tests)

Reply:
OK